app/src/main/res/values/strings.xml

```diff
@@ -33,6 +33,13 @@
     <string name="edit_menu_item">Edit</string>
     <string name="delete_menu_item">Delete</string>
 
+    <string name="activating_bluetooth_went_wrong">Bluetooth could not be activated</string>
+    <string name="activating_bluetooth_went_wrong_explanation">Bluetooth needs to be activated to share public keys over Bluetooth.
```

Comment:
The second line is the explanation, a more insightful sentence about the problem, so a dot at the end is suited.